db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java

```diff
@@ -320,6 +295,11 @@ private int runScriptGuts() {
 				out.flush();
 				command = commandGrabber[currCE].nextStatement();
 
+				if(doSpool) {
```

Comment:
Does this echo the commands to the terminal too? Is that what we want?

Reply:
From the JIRA ticket:
> they asked for a function that would allow them to log sqlshell commands AND results to a file on demand

Let's discuss improvements in a different JIRA maybe?

Reply:
I mean this code seems to be echoing the typed command back to the terminal, so the users would see what they typed twice (once when they type it, once when it's echoed) and then the result. The spooled file would have the command once and the result once, which is correct. I might be wrong and we might not be echoing the command to the terminal again, but I wanted to make sure whether that's the case.

Reply:
Hi Daniel,
sorry I misread you comment, you're right, we're writing the command twice to tty, I will adapt the change accordingly.

Reply:
No worries! Change looks good, thanks!